client/src/app/components/navigation-bar/navigation-bar.component.ts

```diff
@@ -35,9 +36,24 @@ export class NavigationBarComponent implements OnInit {
   facAddIcon = teaseIconPack['facAddIcon'];
   facSortIcon = teaseIconPack['facSortIcon'];
   facCheckIcon = teaseIconPack['facCheckIcon'];
+  facErrorIcon = teaseIconPack['facErrorIcon'];
 
   @Input({ required: true }) allocationData: AllocationData;
 
+  dropdownItems = [
+    { action: this.showExportOverlay.bind(this), icon: this.facExportIcon, label: 'Export', class: 'text-dark' },
```

Comment:
instead of `.bind(this)` you can declare the function as arrow function 
e.g. `showImportOverlay = () => {...} `